modules/http4s-kernel/src/smithy4s/http4s/kernel/package.scala

```diff
@@ -108,7 +108,7 @@ package object kernel {
       .groupBy(_._1)
       .map { case (k, v) => k -> v.map(_._2).toList }
 
-  private[smithy4s] def getRequestMetadata[F[_]](
+  def getRequestMetadata[F[_]](
```

Comment:
**praise:** thanks, I'll be able to remove my copies of these ;)

Reply:
hehe